saleor/graphql/product/filters.py

```diff
@@ -217,9 +218,17 @@ def product_search(phrase):
         phrase (str): searched phrase
 
     """
-    ft_in_description_or_name = Q(search_vector=phrase)
+    query = SearchQuery(phrase, config="english")
```

Comment:
Don't we want to determine the config from `LANGUAGE_CODE` in settings? We would have to obviously map it to values that are accepted by Postgres. That would improve the search for people who run open-source Saleor and use a different language than English.

Reply:
We'll do it separately since currently English is also hardcoded in migrations and supporting more languages is a bit more work.

Reply:
@maarcingebala I think the first proposition is good. I mean, we could use Languade_code to determine the languade, for en we will use English, for different languages we will fallback to the simple config. 
@d-wysocki Is it possible?

Reply:
it's possible but as Marcin said we have hardcoded English in our migrations. For now we would like to stay with this solution as we agreed to hardcode english language at the beginning.

Reply:
Just to confirm. Using simple requires an additional migration?

Reply:
Yes. For now we decided to hardcode English language for search vector.